mozilla-release/browser/base/content/tabbrowser.xml

```diff
@@ -6592,6 +6591,12 @@
         }
       ]]>
       </handler>
+      <handler event="TabRemotenessChange">
```

Comment:
@mikeconley your suggestion worked perfectly. There are couple of things that concern me though.
1. Does the previous `TabParent` die after it's been replaced by another one? If not, then I should probably cleanup the observer from its associated load context.
2. The owner of `TabParent`, the `nsFrameLoader`, swaps it in several situations (`nsFrameLoader::SwapRemoteBrowser`, `nsFrameLoader::SetRemoteBrowser`). It seems, I can still potentially miss the moment, when currently observed `LoadContext` goes away. What do you think about it?